conda/cli/main_env_create.py

```diff
@@ -170,19 +171,19 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
                     result[installer_type] = installer.install(
                         prefix, pkg_specs, args, env
                     )
-                except InvalidInstaller:
+                except InvalidInstaller as exc:
                     raise CondaError(
                         dals(
                             f"""
                             Unable to install package for {installer_type}.
 
                             Please double check and ensure your dependencies file has
                             the correct spelling. You might also try installing the
-                            conda-env-{installer_type} package to see if provides
```

Comment:
Did you know you were supposed to install `conda-env-node` or something like that? o.o

Reply:
Err, TIL!

Reply:
I guess this _was_ planned in the past, but never went anywhere? Seeing https://pypi.org/project/conda-env/ it feels like something went wrong in the past.

Reply:
In the future

conda-env-conda